src/components/expression/expression-loader.js

```diff
@@ -170,6 +169,22 @@ genotet.ExpressionLoader.prototype.loadTfaProfile_ = function(fileName,
 };
 
 /**
+ * Loads expression data list into genotet.data.expressionFiles.
```

Comment:
Yes, it belongs to genotet.data. genotet.data is an object and ExpressionLoaders is one of its keys, as we discussed before.